.circleci/config.yml

```diff
@@ -97,12 +99,60 @@ jobs:
             coverage run --source=super_gradients -m unittest tests/deci_core_unit_test_suite_runner.py
             coverage report
             coverage html  # open htmlcov/index.html in a browser
-
       - store_artifacts:
           path: htmlcov
 
       - store_artifacts:
           path: ~/sg_logs
+      
+  build_and_publish_beta:
+    description : "building the temp image and pushing to dev ecr"
+    parameters:
+       repo_name:
+         type: string
+         default: deci/super-gradients
+       version:
+          type: string
+          default: ${CIRCLE_BRANCH}
```

Comment:
why not tag? so it will be aligned with the RC version of supergradients that we install